cinema/serializers.py

```diff
@@ -1 +1,75 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Movie, Genre, Actor, MovieSession, CinemaHall
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
+        read_only_fields = ["full_name"]
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ["id", "name", "rows", "seats_in_row", "capacity"]
+        read_only_fields = ["capacity"]
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration", "genres", "actors"]
+
+
+class MovieListSerializer(MovieSerializer):
+    genres = serializers.SlugRelatedField(
+        many=True, read_only=True, slug_field="name"
+    )
+    actors = serializers.SlugRelatedField(
```

Comment:
try to use StringRelatedField to remove unnecessary property)

Reply:
I tried to do it, but in such a case I have Key error "full_name" in tests